### DISCUSSION TITLE
Is the bot still limited by the 8MB limit?

### DISCUSSION THREAD

**Discussion by pahaze** [esmBot/esmBot#384] [Q&A] [answered]
As the title asks, I'm wondering if the 8MB limit is still an issue for bots on Discord. When using certain commands on GIFs, some are still uploaded to the external website with the "image is more than 8MB" message. Is the bot actually posting up to 25MB and the message is just wrong, or are bots just unable to use the new 25MB limit yet?

Thanks for your time!

**Comment by samhza** [comment, marked as answer]
bots are able to use the 25MB file limit but esmbot currently isn't using it. the change was made in the source code but the current live version of esmbot hasn't been updated to use the new limit yet

**Reply by pahaze** [reply]
Ah, I see! Thank you!
